### PR TITLE
feat(content): #2496 UX improvements for Pocket

### PR DESCRIPTION
**Pull request by csadilek**
This applies all requested UI changes for the first run of the experiment.

### CODE REVIEW

File: content-src/components/PocketStories/PocketStories.js [hunk 1]
```diff
@@ -71,7 +72,12 @@ const PocketStories = React.createClass({
     return (<section className="pocket-stories spotlight">
       <h3 className="section-title">
         <FormattedMessage id="header_stories" />
-        <span className="pocket-logo" />
+        <span className="section-title-logo" >
+          <span className="pocket-logo-text">
+              <FormattedMessage id="header_stories_by" />
+          </span>
+          <span className="pocket-logo" />
```
Comment (k88hudson) [inline content-src/components/PocketStories/PocketStories.js]: Can you include some text in here for screen readers (just add `<span className="sr-only">Pocket</span>` as a child of these element)

**Comment by k88hudson** [comment]
Can you also add a note to the port ticket (https://github.com/mozilla/activity-stream/issues/2512) to fix the RTL issue with the `>` character?

File: content-src/components/PocketStories/PocketStories.js (hunk 1 above)
Comment (csadilek) [inline content-src/components/PocketStories/PocketStories.js]: Thanks @k88hudson, changes applied.

**Comment by as-pine-proxy** [comment]
[Treeherder: a36510daf79edc4471f673b8df4f2891a01d061b](https://treeherder.mozilla.org/#/jobs?repo=pine&revision=a36510daf79edc4471f673b8df4f2891a01d061b)

**Pull request merged**